adaezeodurukwe/iReport
# #163053373 Build signup page

**What does this PR do?**
 Makes the signup page functional

**Description of tasks to be completed?**
- Add signup page
- Add signup script
- Enable cors for iReporter app

**How should this be tested?**
After cloning the repo, check out to ft-build-signup-frontend-163053373 branch and run the signup.html file in the client folder.

- Enter all required details and click the "Sign up" button

**What are the relevant pivotal tracker stories?**
#163053373
